packaging/nuget/nservicebus.host/EndpointConfig.cs

```diff
@@ -7,9 +7,17 @@ public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(EndpointConfiguration endpointConfiguration)
         {
-            // NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence. 
+            //TODO: NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence.
             // Refer to the documentation for more details on specific options.
             endpointConfiguration.UsePersistence<PLEASE_SELECT_ONE>();
+
+            //TODO: NServiceBus will move messages that fail repeatedly to a separate "error" queue. We recommend 
+            // that you start with a shared error queue for all your endpoints for easy integration with ServiceControl.
+            endpointConfiguration.SendFailedMessagesTo("error");
+
+            //TODO: NServiceBus will store a copy of each successfully process message in a separate "audit" queue. We recommend
```

Comment:
@bording makes a good point, the values are appropriate for all transports other than msmq. And for msmq they are ok for local development, users will find out soon enough, and when they do there is no message loss.

I'm :+1: to remove the todos?